bigquery_etl/cli/query.py

```diff
@@ -740,17 +769,14 @@ def backfill(
             "date_partition_parameter", "submission_date"
         )
         date_partition_offset = metadata.scheduling.get("date_partition_offset", 0)
+        scheduling_parameters = metadata.scheduling.get("parameters", [])
 
-        # For backwards compatibility assume partitioning type is day
-        # in case metadata is missing
-        if metadata.bigquery:
+        partitioning_type = None
+        if metadata.bigquery and metadata.bigquery.time_partitioning:
             partitioning_type = metadata.bigquery.time_partitioning.type
-        else:
-            partitioning_type = PartitionType.DAY
-            click.echo("Bigquery partitioning type not set. Using PartitionType.DAY")
 
         match partitioning_type:
-            case PartitionType.DAY:
+            case None | PartitionType.DAY:
```

Comment:
Even if the table isn't partitioned, most queries still run daily, so format the dates in the same way